server/cluster/coordinator.go

```diff
@@ -155,6 +160,49 @@ func (c *coordinator) patrolRegions() {
 	}
 }
 
+// checkMissRegions check miss peer ,if miss region is more than maxMissPeerQueueSize,
+// it will return ture to avoid run other replica check
+func (c *coordinator) checkMissRegions() bool {
+	add := make([]*cache.Entry, 0)
+	remove := make([]uint64, 0)
+	missPeers := c.checkers.GetMissRegions()
+	regionListGauge.WithLabelValues("priority").Set(float64(len(missPeers)))
+	log.Debug("check miss regions", zap.Int("miss region count", len(missPeers)))
+	for _, entry := range missPeers {
+		id, ok := entry.Value.(uint64)
+		if !ok {
+			continue
+		}
+		region := c.cluster.GetRegion(id)
+		if region == nil {
+			// the region could be recent split, continue to wait.
+			remove = append(remove, id)
+			continue
+		}
+		if c.opController.GetOperator(id) != nil {
```

Comment:
I think we don't need to check it here.

Reply:
yes, it will block by low priority operator like merge